src/patterns/organisms/hero/hero-branded.hbs

```diff
@@ -0,0 +1,33 @@
+---
+name: A Firefox branded hero with image
+order: 3
+description: |
+  A hero section with Firefox product branding. This example also features a tagline.
+notes: |
+  - Adding a theme class of `mzp-t-firefox` will change the title font to Open Sans and add a default curve shape to the bottom.
+  - Product icons can be displayed using an additional theme class:
+    - `mzp-t-product-firefox`.
+    - `mzp-t-product-beta`.
+    - `mzp-t-product-developer`.
+    - `mzp-t-product-nightly`.
+    - `mzp-t-product-focus`.
+  - The bottom curve shape can be swapped with a different shape in your page’s CSS (it’s an SVG mask). You can also change the shape’s fill color in CSS with the selector `.mzp-c-hero.mzp-t-firefox:after`
+  - The page section immediately after/below the hero should have a background color matching the color of the curve shape so the sections will merge seamlessly.
```

Comment:
Is this something that may be worth doing in the demo page, to show an example of intended usage? The abrupt end to the curves looks a little odd.

Reply:
Good call. I'll add some filler blocks to demonstrate the effect. (I did have some at one point; the demo CSS even still has the class because I was sloppy 😏 ).